server/src/services/album.service.ts

```diff
@@ -245,6 +247,8 @@ export class AlbumService {
       }
 
       await this.albumUserRepository.create({ userId: userId, albumId: id, role });
+
+      await this.jobRepository.queue({ name: JobName.NOTIFY_ALBUM_INVITE, data: { album, guestUser: user } });
```

Comment:
All other jobs just pass ids that are double checked in the job implementation. I would probably prefer that here. So just send the albumId and the userId to send the email to instead of the whole entity.